projects/llm-semconv.md

```diff
@@ -0,0 +1,72 @@
+# LLM Observability Semantic Conventions
+
+## Description
+
+This project aims to establish observability standards for Large Language Model (LLM) applications, focusing on API interactions and multi-step LLM frameworks. Key areas include:
+
+- Developing semantic conventions for standard and complex LLM interactions.
+- Initiating vendor-specific conventions as a foundation for broader contributions.
+- Concentrating on attributes relevant to Metrics and Spans.
+
+For Spans: Instrumentation and debugging guidance for multi-step LLM applications, with recommendations on user privacy and intellectual property security, marking sensitive attributes and events as opt-in.
+
+For Metrics: Tracking model performance by collecting and correlating data like model names, versions, parameters, latency, and evaluation scores.
+
+## Deliverables
+
+- Experimental semantic conventions for LLMs, allowing flexibility for evolution in this fast-paced field.
+- Reference prototypes demonstrating the application of these conventions in data collection, processing, and analysis.
+- Guidelines for monitoring LLM applications, including implementation strategies and privacy/security considerations.
+
+## Staffing / Help Wanted
+
+We will gather further staff as the project is established. We need to 
+make sure there is enough representation across both vendors and customers
+who will benefit from these semantic conventions.
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- *Seeking additional sponsors*
+
+**Engineers:**
+
+- @nirga
+- @sudivate
+- *Positions open for more engineers*
+
+**Maintainers:**
+
+- To be determined
+
+## Meeting Times
+
+Alternating weekly meetings to accommodate different time zones:
+
+- Week 1: Mondays at 8 AM PST
```

Comment:
this conflicts with OTel SemConv meeting and it's unlikely I and other people involved into OTel semconv will be able to make this time work.  We can tentatively keep it and see how it works though.

Reply:
Updated the time for later in the week.